src/test/java/GoogleSearchTest.java

```diff
@@ -0,0 +1,41 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class GoogleSearchTest {
+
+    private WebDriver driver;
+
+    @BeforeClass
+    public static void setupWebDriver() {
+        System.setProperty("webdriver.chrome.driver",System.getProperty("user.dir")+"/src/main/resources/chromedriver.exe");
```

Comment:
dont  upload the driver, check this library https://github.com/bonigarcia/webdrivermanager